flink-rpc/flink-rpc-core/src/main/java/org/apache/flink/runtime/rpc/RpcUtils.java

```diff
@@ -41,9 +41,10 @@
 public class RpcUtils {
 
     /**
-     * <b>HACK:</b> Set to 21474835 seconds, Akka's maximum delay (Akka 2.4.20). The value cannot be
-     * higher or an {@link IllegalArgumentException} will be thrown during an RPC. Check the private
-     * method {@code checkMaxDelay()} in {@link akka.actor.LightArrayRevolverScheduler}.
+     * <b>HACK:</b> Set to 21474835 seconds, Pekko's maximum delay (Akka 2.4.20). The value cannot
+     * be higher or an {@link IllegalArgumentException} will be thrown during an RPC. Check the
+     * private method {@code checkMaxDelay()} in {@link
+     * org.apache.pekko.actor.LightArrayRevolverScheduler}.
```

Comment:
```suggestion
     * org.apache.pekko.actor.LightArrayRevolverScheduler}.
```
That's odd: My Intellij isn't able to resolve the reference even though it's the correct location. :thinking: It even suggests this class name when trying to fix the error

Reply:
Expected because it's not a public class.